lib/tasks/configuration.rake

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require_relative 'task_helpers'
+
+namespace :configuration do
+  desc 'Create an administrator account'
+  task check: :environment do
+    required_env_vars = %w[SECRET_KEY_BASE BIGBLUEBUTTON_ENDPOINT BIGBLUEBUTTON_SECRET DATABASE_URL REDIS_URL].freeze
+
+    # Initial check that variables are set
+    info 'Checking required environment variables:'
+    required_env_vars.each do |var|
+      failed("#{var} not set correctly") if ENV[var].blank?
+    end
+    passed
+
+    info 'Checking connection to Postgres Database:'
+    begin
+      ActiveRecord::Base.establish_connection # Establishes connection
+      ActiveRecord::Base.connection # Calls connection object
+      failed('Failed to connect to Database') unless ActiveRecord::Base.connected?
+    rescue StandardError => e
+      failed("Failed to connect to Database - #{e}")
+    end
+    passed
+
+    info 'Checking connection to Redis Cache:'
+    begin
+      Redis.new.ping
+    rescue StandardError => e
+      failed("Failed to connect to Redis - #{e}")
+    end
+    passed
+
+    info 'Checking connection to BigBlueButton:'
+    test_request(Rails.configuration.bigbluebutton_endpoint)
+    checksum = Digest::SHA1.hexdigest("getMeetings#{Rails.configuration.bigbluebutton_secret}")
+    test_request("#{Rails.configuration.bigbluebutton_endpoint}getMeetings?checksum=#{checksum}")
+    passed
+
+    if ENV['SMTP_SERVER'].present?
+      info 'Checking connection to SMTP Server'
+      begin
+        UserMailer.with(to: ENV.fetch('SMTP_SENDER_EMAIL', nil), subject: ENV.fetch('SMTP_SENDER_EMAIL', nil)).test_email.deliver_now
```

Comment:
I think we're not enabling raising exceptions for failed emails delivery?

Reply:
We may want to indicate on success to admins to check the inbox folder of the `SMTP_SENDER_EMAIl` address?

Reply:
It will be mentioned in the docs that about the email that will be sent to SMTP_SENDER_EMAIl

Reply:
Yep